nilearn/plotting/tests/test_img_plotting.py

```diff
@@ -52,6 +53,32 @@ def demo_plot_roi(**kwargs):
     return plot_roi(img, title="Broca's area", **kwargs)
 
 
+def test_plot_roi_view_types():
+    # This is only a smoke test contours rois
+    demo_plot_roi(view_type='contours')
+    # This is only a smoke test contours rois
+    demo_plot_roi(view_type='continuous')
+
+    # Test error message for invalid view_type
+    assert_raises_regex(ValueError,
+                        'Unknown view type:',
+                        demo_plot_roi, view_type='flled')
+    plt.close()
+
+
+def test_plot_roi_contours():
+    display = plot_roi(None)
+    data = np.zeros((91, 109, 91))
+    x, y, z = -52, 10, 22
+    x_map, y_map, z_map = coord_transform(x, y, z,
+                                          np.linalg.inv(mni_affine))
+    data[int(x_map) - 5:int(x_map) + 5, int(y_map) - 3:int(y_map) + 3,
+         int(z_map) - 10:int(z_map) + 10] = 1
+    img = nibabel.Nifti1Image(data, mni_affine)
+    _plot_roi_contours(display, img, 'RdBu', 0.1, 2.)
```

Comment:
I think that I would prefer it if you used the public-facing function in the test.

Reply:
@KamalakerDadi I think this is the last comment to address before this is merged?

> I think that I would prefer it if you used the public-facing function in the test.